client/Components/GameBoard/ActivePlayerPrompt.jsx

```diff
@@ -131,8 +131,8 @@ class ActivePlayerPrompt extends React.Component {
         }
 
         let timer = null;
-
-        let promptText = this.props.promptText ? (this.props.promptText.text || this.props.promptText) : null;
```

Comment:
this could be:

let promptText = this.props.promptText && this.props.promptText.text ? this.props.promptText.text : this.props.promptText;

Reply:
Make the code more explicit now